modules/cnf-programming-the-n3000-operator.adoc

```diff
@@ -0,0 +1,351 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="programming-the-openness-operator-for-intel-fpga-pac-n3000-operator_{context}"]
+= Programming the OpenNESS Operator for Intel® FPGA PAC N3000
+
+This section describes how to program the Intel® FPGA PAC N3000 with a vRAN 5G bitstream. This bitstream exposes the Single Root I/O Virtualization (SR-IOV) virtual function (VF) devices used to accelerate the FEC in the vRAN workload.
```

Comment:
Instead of the self-referential "This section...", I think you can use "As an administrator, you can program the Intel FPGA PAC..."

s/devices used/devices that are used/